### ISSUE TITLE
Change on behavior - sem-apply not applying the migrations if run in non interactive scripts

### ISSUE THREAD

**Issue by koldLight** [mbryzek/schema-evolution-manager#78]
Hi there,

First of all, thanks a lot for this repo. It's my favorite choice as a simple tool for applying migrations in all my projects.

Last week I noticed a change on behavior. I'm running always the "sem apply" in my clients docker to run all the migrations, and then I start the services (e.g. backend, etc). It always run OK since last week it stopped applying this migrations, because it prompts the following message to the users:

```
Please confirm that you would like to apply all (<n>) of the pending scripts:
...
```

As it's running inside a docker an non interactive way, it ignores the migrations and continue with the script.

I think it's related to #76 . It would be better if the default value of the `non_interactive` parameter were 'true' so that the behavior remains the same before and after the change.. If you want, I can do a PR for this.

Thanks,

**Comment by sergio-dtm** [comment]
Hi, We are facing this same issue. Our automated deployment processes stoped working due to the added interactivity of sem. I think it should be non interactive by default as well.

Thanks for this useful tool.

Regards.

**Comment by mbryzek** [comment]
@koldLight @sergio-dtm Thank you for your notes and my apologies for the regression. I've reverted the change and published a new gem (version 0.9.54).

We'll rethink the approach on the `non_interactive` feature on a new PR including your comments on backwards compatibility.
